services/core/cards-publication/src/main/modeling/swagger.yaml

```diff
@@ -350,6 +350,21 @@ definitions:
         type: array
         items:
           type: string
+  SubscriptionManagement:
+    type: object
+    description: Objet containing a list of active subscriptions 
+    properties:
+      subscriptions:
+        type: array
+        items:
+          $ref: '#/definitions/SubscriptionDescription'
+  SubscriptionDescription:
```

Comment:
Actually after testing with duplicate connections with the same user I think I can see why we need the Subscription description object... But shouldn't we call it just "Subscription" as it's the resource we interact with?
I mean let's imagine that in the future we can call this endpoint to forcibly close a subscription, it would be something like "DELETE subscriptionManagement/subscription/{subscription_id}" not "DELETE subscriptionManagement/subscriptionDescription/{subscription_id}"

To be honest after writing "subscription" about 20 times in this review I think we should never have called them subscriptions in the first place because IT IS SUCH AN ANNOYING WORD TO SPELL ;-)

Reply:
proposal : naming it connection ?

Reply:
New endpoint /connections done